pkgs/top-level/all-packages.nix

```diff
@@ -1145,19 +1145,21 @@ with pkgs;
           fetchurl = stdenv.fetchurlBoot;
           inherit zlib openssl;
         };
-        # On darwin, libkrb5 needs bootstrap_cmds which would require
+        # On darwin, krb5 needs bootstrap_cmds which would require
         # converting many packages to fetchurl_boot to avoid evaluation cycles.
         # So turn gssSupport off there, and on Windows.
         # On other platforms, keep the previous value.
         gssSupport =
           if stdenv.isDarwin || stdenv.hostPlatform.isWindows
             then false
             else old.gssSupport or true; # `? true` is the default
-        libkrb5 = buildPackages.libkrb5.override {
+        krb5 = (buildPackages.krb5.override {
           fetchurl = stdenv.fetchurlBoot;
           inherit pkg-config perl openssl;
+          withLibedit = false;
+          byacc = buildPackages.byacc.override { fetchurl = stdenv.fetchurlBoot; };
           keyutils = buildPackages.keyutils.override { fetchurl = stdenv.fetchurlBoot; };
-        };
+        }).lib;
```

Comment:
Probably needs `dev` too?

Reply:
It certainly doesn't *need* it, but I don't think there's a good reason to pick this output anymore. I'll get rid of this `.lib` part.